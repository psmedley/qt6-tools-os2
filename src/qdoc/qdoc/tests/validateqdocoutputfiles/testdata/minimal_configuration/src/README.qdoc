// Copyright (C) 2024 The Qt Company Ltd.
// SPDX-License-Identifier: LicenseRef-Qt-Commercial OR GFDL-1.3-no-invariants-only

/*!
    \page readme.html
    \title README

    This test is for a minimal .qdocconf file.
*/
