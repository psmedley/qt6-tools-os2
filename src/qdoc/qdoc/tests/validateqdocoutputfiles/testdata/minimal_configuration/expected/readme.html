<!DOCTYPE html>
<html lang="en">
<head>
  <meta charset="utf-8">
<!-- README.qdoc -->
  <title>README | A minimal QDoc configuration</title>
</head>
<body>
<div class="sidebar"><div class="sidebar-content" id="sidebar-content"></div></div>
<h1 class="title">README</h1>
<!-- $$$readme.html-description -->
<div class="descr" id="details">
<p>This test is for a minimal .qdocconf file.</p>
</div>
<!-- @@@readme.html -->
</body>
</html>
